### ISSUE TITLE
[Bug]:  Can't overwrite whichkey binding for nvim tree

### ISSUE THREAD

**Issue by carvhal** [LunarVim/LunarVim#1755]
### Problem description

In config.lua:

` lvim.builtin.which_key.mappings["e"] = { "<cmd>RnvimrToggle<CR>", "Ranger" }`

Does not work, the default keybind `"<cmd>NvimTreeToggle<CR>", "Explorer"` is not overwritten. 
I noticed that this binding is defined in `nvimtree.lua` while most keybinds are defined in `which-key.lua`. Moving this definition to `which-key.lua` fixes the problem.

It seems that there are a few other default which key bindings being defined outside `which-key.lua`, namely dashboard and dap bindings.

### LunarVim version

rolling

### Neovim version (>= 0.5)

NVIM v0.5.1. LuaJIT 2.0.5

### I have

- [X] Read the readme
- [X] Read the wiki
- [X] Searched for similar issues

**Comment by ssbanerje** [comment]
See #1648.

**Comment by abzcoding** [comment]
this is fixed via #1965

**Comment by danilobjr** [comment]
@abzcoding 

I didn't open a new issue because this one is the exact problem I'm facing now.
So, I saw that after #1965 we can unbind NvimTree with the following:

```
lvim.keys.normal_mode["<space>e"] = false
``` 

This works, but when I hit `<space>`, which_key shows:

![Screen Shot 2022-08-06 at 21 57 27](https://user-images.githubusercontent.com/735556/183270691-59eafa12-dac7-4107-87e7-ee4b3d728720.png)

And even if I set which_key with a different binding, it takes no effect.

What I'm trying to do is:

```lua
lvim.builtin.which_key.mappings["e"] = {
  name = "+Editor",
  e = { "<cmd>NvimTreeToggle<cr>", "Explorer" },
  -- I'd like to set other bindings here
}
```

Maybe `lvim.builtin.which_key.mappings["e"] = { "<cmd>NvimTreeToggle<CR>", "Explorer" }` should be moved from nvimtree.lua to which_key.lua? I've tried this approach locally and it works properlly (after restart).

**Comment by nikbrunner** [comment]
Same problem here. Doesnt matter what I do `<leader>e` always maps to "Explorer".

What I tried:
```lua
local wk = lvim.builtin.which_key

wk.mappings["e"] = {}
wk.mappings.e = {
    name = "Edit",
    l = { cmd.edit.log_word, "Log Word" },
    d = { cmd.edit.delete_logs, "Delete Logs" },
    u = { cmd.edit.undo, "UndoTree" },
}
```
